### PR TITLE
Remove the configuration of synchronous mode from init_cfg.json

### PR DESCRIPTION
**Pull request by shi-su**
**- Why I did it**
Remove the configuration of synchronous mode from init_cfg.json

**- How I did it**
Remove the configuration of synchronous mode from init_cfg.json by removing it from the template file.

**- How to verify it**
I verified it by testing locally with a virtual switch.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -573,7 +574,9 @@ def parse_meta(meta, hname):
                     region = value
                 elif name == "CloudType":
                     cloudtype = value
-    return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id, region, cloudtype
+                elif name == "SynchronousMode":
```
Comment (qiluo-msft) [inline src/sonic-config-engine/minigraph.py:577]: >SynchronousMode [](start = 30, length = 15)

The input minigraph.xml has no new field for you to parse.

File: slave.mk [hunk 2]
```diff
@@ -230,7 +230,6 @@ $(info "INCLUDE_SFLOW"                   : "$(INCLUDE_SFLOW)")
 $(info "INCLUDE_NAT"                     : "$(INCLUDE_NAT)")
 $(info "INCLUDE_KUBERNETES"              : "$(INCLUDE_KUBERNETES)")
 $(info "TELEMETRY_WRITABLE"              : "$(TELEMETRY_WRITABLE)")
-$(info "ENABLE_SYNCHRONOUS_MODE"         : "$(ENABLE_SYNCHRONOUS_MODE)")
```
Comment (qiluo-msft) [inline slave.mk:233]: >ENABLE_SYNCHRONOUS_MODE [](start = 8, length = 23)

Just wondering if there is a need to build for async/sync for community members.

File: files/build_templates/sonic_debian_extension.j2 [hunk 3]
```diff
@@ -538,6 +538,9 @@ sudo LANG=C cp $SCRIPTS_DIR/radv.sh $FILESYSTEM_ROOT/usr/local/bin/radv.sh
 # Copy sonic-netns-exec script
 sudo LANG=C cp $SCRIPTS_DIR/sonic-netns-exec $FILESYSTEM_ROOT/usr/bin/sonic-netns-exec
 
+# Copy sonic-sync-mode script
+sudo LANG=C cp $SCRIPTS_DIR/sonic-sync-mode $FILESYSTEM_ROOT/usr/bin/sonic-sync-mode
```
Comment (qiluo-msft) [inline files/build_templates/sonic_debian_extension.j2:542]: >sonic-sync-mode [](start = 69, length = 15)

Does it make more sense to move this script to sonic-utilities repo? If yes, you don't need to manually install.

File: files/scripts/sonic-sync-mode [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import argparse
+from swsscommon import swsscommon
+
+def main():
+    parser=argparse.ArgumentParser(description=
+          'Configure enable or disable synchronous mode between orchagent and syncd',
+          formatter_class=argparse.RawTextHelpFormatter,
+          epilog=
+'''
+**sudo** needed for configuring synchronous mode
```
Comment (qiluo-msft) [inline files/scripts/sonic-sync-mode:11]: >**sudo** [](start = 0, length = 8)

Do you mean bold font? It's difficult to read.

**Review by qiluo-msft — CHANGES_REQUESTED** [review]
As comments

File: src/sonic-config-engine/minigraph.py (hunk 1 above)
Comment (shi-su) [reply src/sonic-config-engine/minigraph.py]: Removed the field parser in minigraph .xml

File: slave.mk (hunk 2 above)
Comment (shi-su) [reply slave.mk:233]: I feel it is not a must as long as we provide the community with the CLI to change the configuration. The users can update the mode after booting up the image, and this scenario should be pretty rare.

File: files/build_templates/sonic_debian_extension.j2 (hunk 3 above)
Comment (shi-su) [reply files/build_templates/sonic_debian_extension.j2]: Moved it to sonic-utilities repo.

File: files/scripts/sonic-sync-mode (hunk 4 above)
Comment (shi-su) [reply files/scripts/sonic-sync-mode]: Fixed this and moved the CLI to sonic-utilities repo.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**